README.md

```diff
@@ -48,3 +48,14 @@ aws s3 cp path/to/some_cards.json s3://decision-support-tool-app-dev/
 
 Replace `<ENVIRONMENT>` with your environment, e.g., `dev`.
 Note the arguments `"dataset_identifier", "SNAP", "Michigan", "s3://decision-support-tool-app-dev/some_cards.json"` are in the same order as described above for `ingest-guru-cards`, i.e., `DATASET_ID BENEFIT_PROGRAM BENEFIT_REGION FILEPATH`.
+
+
+### Web scraping
+
+Scrape web pages and save to a JSON file, for example: 
+```sh
+cd app/src/ingestion
+python scrape_edd_web.py
```

Comment:
(This is also important to make sure the command is run with poetry run... so that the virtual environment is activated, otherwise devs need the dependencies installed globally)

Reply:
Done